src/workerd/util/sqlite.c++

```diff
@@ -326,6 +326,7 @@ enum class PragmaSignature {
   NO_ARG,
   BOOLEAN,
   OBJECT_NAME,
+  NO_ARG_OR_OBJECT_NAME,
```

Comment:
My first reading of this, I interpreted it as "no (arg or object name)", i.e. "no arg and also no object name".

Maybe `OPTIONAL_OBJECT_NAME` would be clearer?